setup.cfg

```diff
@@ -1,2 +1,6 @@
 [flake8]
-ignore=E501
+# Ignore:
+#
+#   N812 - We like doing EC for the expected_conditions, even 
```

Comment:
In our code  we have lines like:

    from selenium.webdriver.support import expected_conditions as EC

I talked with Blair and he said that the selenium docs suggest doing that (it actually in my opinion makes the code compact yet readable).   However I could see just using expected_conditions (cause that's really readable albeit long), but then our code would be idiomatically different (I think that is the right word) than much other selenium code out there.   As I told Blair, I could go either way...but if we stayed this way, I should put some of this information in that comment.

And I don't know what E501 is for, Blair added that (-;   Let me look that up...it's the max line length.   I actually prefer lines to be under 80, but sometimes they need to be longer.  I'll add a comment.